.clang-tidy

```diff
@@ -189,4 +189,4 @@ CheckOptions:
   - { key: readability-function-size.LineThreshold,               value: 200 }
   - { key: readability-function-size.StatementThreshold,          value: 200 }
   - { key: readability-function-size.BranchThreshold,             value: 10 }
-  - { key: readability-function-size.ParameterThreshold,          value: 4 }
+  - { key: readability-function-size.ParameterThreshold,          value: 5 }
```

Comment:
I had to change the parameter threshold because of the `LogStream` c'tor with 5 parameters.

Reply:
Why didn't clang-tidy nag about this before? I think it would be better to just add a clang-tidy suppression in that file instead of increasing the threshold. It will be fixed once the `SourceLocation` struct is available

Reply:
I don't know why clang-tidy didn't complain before, especially because I just saw that it already complained about another `LogStream` c'tor. So I copied the suppression.